core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4233,6 +4233,40 @@ private void checkLiteral2(String expression, String expected) {
     });
   }
 
+  @Test public void testCurrentUser() {
+    String query = "select CURRENT_USER from \"product\" where \"product_id\" = 1";
+    final String expectedSql = "SELECT CURRENT_USER() CURRENT_USER\n"
```

Comment:
Does CURRENT_USER as alias work on hive and spark without identifier quotes???

Reply:
=> For hive, current_user is working fine.
=> For spark current_user we are getting below exception:Error: org.apache.spark.sql.AnalysisException: Undefined function: 'current_user'. This function is neither a registered temporary function nor a permanent function registered in the database 'default'.; line 1 pos 7 (state=,code=0)
Hence removed spark current_user changes.